WORKSPACE

```diff
@@ -121,32 +121,36 @@ load(
     "container_pull",
 )
 
+# Pulled gcr.io/distroless/cc-debian11:latest on 2022-02-23
```

Comment:
Reviewers: I know today's date is Feb 28, but these images were created on the 23rd so that is why i referenced it this way.